packages/create-next-app/templates/app-api/js/app/with-middleware.js

```diff
@@ -0,0 +1,61 @@
+import { NextRequest, NextResponse } from "next/server";
```

Comment:
In this file, we're not using `NextRequest` . Is there any reason to import it here?

Reply:
Removed. Thanks.